src/test/java/org/ays/auth/controller/AdminRegistrationApplicationControllerTest.java

```diff
@@ -768,4 +768,49 @@ void givenValidAdminRegisterApplicationRejectRequest_whenUnauthorizedForRejectin
                 .reject(Mockito.eq(mockId), Mockito.any(AdminRegistrationApplicationRejectRequest.class));
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = {
+            "",
+            "   ",
+            "less than 40",
+            """
+                    Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.
+                    Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+                    Donec qudam felis, ultricies nec, pellentesque eu, pretscsxwium quis, sem. Nulla consequat massa quis
+                    enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut,
+                    imperdiet a, venenatdskjvndshjcndsis vitae, justo. Nullam dictum felis eu pedde mollis pretium. Integer tincidunt.
+                    Cras dapibus. Vivdamus ewl
+                    """,
+            " spaceAtTheBeginning",
+            "spaceAtTheEnd ",
+            " both ",
+            "   justAString     "
+    })
+    void givenValidAdminRegisterApplicationRejectRequest_whenRejectingAdminRegisterApplication_thenReturnValidationError(String reason) throws Exception {
```

Comment:
Test metodu ismi, hatalı veya uygunsuz red gerekçelerini kontrol eden bir testi tanımladığı için valid yerine invalid yazmak daha uygun olur gibi

Reply:
tesekkurler, duzelttim